app/services/feature_toggles/bundle.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+module FeatureToggles
```

Comment:
I like putting all this in a module outside of the controller, seems cleaner overall